sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/ADIS16448.scala

```diff
@@ -9,7 +9,7 @@ import squants.motion.{AngularVelocity, DegreesPerSecond}
 /**
   * An interface for communicating with the ADIS16448 IMU.
   */
-class ADIS16448(spi: SPITrait, updatePeriod: Time) extends DigitalGyro(updatePeriod) {
+class ADIS16448(spi: SPITrait, updatePeriod: Time, maxDriftDeviation: Value3D[AngularVelocity]) extends DigitalGyro(updatePeriod, maxDriftDeviation) {
```

Comment:
Instead of making this a constructor parameter here and in `DigitalGyro`, can you make it a parameter to the check method?